CocoaPods/Podfile.lock

```diff
@@ -208,53 +206,8 @@ DEPENDENCIES:
 
 SPEC REPOS:
   https://repositories.tomtom.com/artifactory/api/pods/cocoapods:
-    - TomTomSDKBindingFrameworkHTTPInternal
```

Comment:
I am not sure, why those dependencies are gone?

Reply:
This is an update to the spec repository section in the lockfile which is changed as part of using new version of cocoapods. The pods and dependencies are still listed as required in the lines above.